.github/PULL_REQUEST_TEMPLATE/pull_request_new-model.md

```diff
@@ -0,0 +1,11 @@
+<!-- Describe your new model briefly here -->
+
+
+
+# New model TODOs
+
+- [ ] Add default hyperparameter file to `models/cli/conf/architecture/<YOUR_model>.yaml`
+- [ ] Add a prediction test?
+- [ ] Add a fitting test?
```

Comment:
Torch-scriptability is required for experimental models. Then I would say
```suggestion
- [ ] Tests: torch-scriptability, basic functionality (invariance, fitting, prediction)
```

Reply:
Good point!